presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataReader.java

```diff
@@ -133,7 +133,8 @@ public Footer readFooter(HiveWriterVersion hiveWriterVersion, InputStream inputS
                 toStripeInformation(footer.getStripesList()),
                 toType(footer.getTypesList()),
                 toColumnStatistics(hiveWriterVersion, footer.getStatisticsList(), false),
-                toUserMetadata(footer.getMetadataList()));
+                toUserMetadata(footer.getMetadataList()),
```

Comment:
do you have pending commit to extract EncryptionGroup from orc footer and wire it up here?

Reply:
No, for now the plan is just to add support for encrypted DWRF, and not encrypted ORC.